spec/features/letters/view_letter_preview_spec.rb

```diff
@@ -0,0 +1,119 @@
+require 'rails_helper'
+
+describe 'Viewing A Letter Preview' do
+  let(:uuid) { SecureRandom.uuid }
+  let(:preview) { Faker::DumbAndDumber.quote }
+  let(:document_id) { Faker::Number.between(1, 1_000) }
+
+  before do
+    create_jwt_token
+  end
+
+  context 'when sending a rents letter' do
+    before do
+      stub_my_cases_response
+      stub_get_templates_response
+      stub_success_post_send_letter_response
+    end
+
+    scenario do
+      given_i_am_logged_in
+      when_i_visit_new_letter_page
+      then_i_see_a_letter_form
+    end
+
+    scenario do
```

Comment:
We should really add what the scenario is.

```suggestion
    scenario 'Redirect when the user refreshes the Preview Page' do
```